### DISCUSSION TITLE
Alternate use for a PowR2

### DISCUSSION THREAD

**Discussion by eddee54455** [arendst/Tasmota#10169] [Show and tell]
Hi guys...

In the spirit of "Doing it Different" ...

I use a couple of PowR2 switches flashed with Tasmota to monitor the power produced by a couple of small, cheap, Grid Tie Inverters....

Works quite well, and in doing so, I am able to extrapolate what my total available solar yield can be on my main, Axpert inverter...

Based on the extrapolation, I switch additional loads in as solar becomes available... All controlled, monitored and forecast by my NodeRed and linked via MQTT.... All on a Pi4!

Cheerz
Ed

**Comment by Moosbueffel** [comment]
Why didn't you watch your main Axpert inverter??
And how did you the optimizing of the PV power? Do you use a SCR, or add simple heater one after the other?

**Reply by eddee54455** [reply]
I do watch the Ax output, but that only tells what it is supplying, not the GTI's that are connected after it....

Based on the GTI output, it is giving as much as it can, the Ax can supply additional to a max as extrapolated...

For the optimizing, I use what power currently forecast as available, then juggle loads to use the most possible... Everything from dishwashers, fridges, freezers, Hot water boiler(8 stage element installed) .... All controlled by and large by Tasmotized Sonoffs

E
